akka-remote/src/main/scala/akka/remote/artery/compress/CompressionSettings.scala

```diff
@@ -13,21 +13,18 @@ import scala.concurrent.duration._
 /** INTERNAL API */
 private[akka] class CompressionSettings(_config: Config) {
   val enabled = _config.getBoolean("enabled")
-  @inline private def globalEnabled = enabled
 
   val debug = _config.getBoolean("debug")
 
   object actorRefs {
     private val c = _config.getConfig("actor-refs")
 
-    val enabled = globalEnabled && c.getBoolean("enabled")
```

Comment:
remove the flag in some string configs as well, AFAIR MaxThroughputSpec also sets this to `on` explicitly